src/option.h

```diff
@@ -98,9 +98,34 @@ namespace librealsense
      protected:
         const option_range _opt_range;
         std::function<void(const option&)> _recording_function = [](const option&) {};
+    };
+
+    class option_with_description : public virtual option
```

Comment:
This is undesired - the functionality of adding description should be retrofitted to the base class as optional.
To be refactored